apps/src/gamelab/GameLab.js

```diff
@@ -171,14 +183,18 @@ GameLab.prototype.init = function (config) {
     });
   }.bind(this);
 
-  ReactDOM.render(React.createElement(AppView, {
+  this.reduxStore_.dispatch(actions.setInitialLevelProps({
     assetUrl: this.studioApp_.assetUrl,
     isEmbedView: !!config.embed,
-    isShareView: !!config.share,
-    renderCodeWorkspace: renderCodeWorkspace,
-    renderVisualizationColumn: renderVisualizationColumn,
-    onMount: onMount
-  }), document.getElementById(config.containerId));
+    isShareView: !!config.share
+  }));
+
+  ReactDOM.render(<Provider store={this.reduxStore_}>
+    <GameLabView
+      renderCodeWorkspace={renderCodeWorkspace}
+      renderVisualizationColumn={renderVisualizationColumn}
+      onMount={onMount} />
```

Comment:
For now, these are functions passed around explicitly for the purpose of doing one-time non-React EJS rendering _into_ `ProtectedStatefulDiv`s.  If they ever return React components, we probably don't need to pass them as props anymore.

I'll come up with a name that makes it more obvious what these are for.